applications/CipherEX.md

```diff
@@ -0,0 +1,82 @@
+# Open Grant Proposal
+* **Project:** CipherEX
+* **Proposer:**  Pawn
+* **Payment Address:**  DAI: 0x450b7251B9c1dBA8b16056558E445F8bc179Cc11
+## Project Overview :page_facing_up:
+### Overview
+This project deliver a privacy preserving data sharing pallet(CipherEX) based on proxy re-encryption technology in Polkadot Ecosystem. 
+
+ If someone want to send a secret data to another, the tradition solution is using Diffie-Hellman Key exchange to generate a shared key and then they can encode the secret data using the shared key with symmetric encryption scheme such AES. 
+
+Instead of  Diffie-Hellman Key exchange, in our proposal we will use a threshold proxy re-encryption technology to transport the symmetric key which used for symmetric encryption. The idea comes from this [paper](https://github.com/nucypher/umbral-doc/blob/master/umbral-doc.pdf) .  And based on this ,we could design a "fair exchange" of secret data under a "Proxy-exist" setting.
```

Comment:
There is nothing here about why threshold proxy reencryption works better than [FairSwap](https://eprint.iacr.org/2018/740.pdf) or [CMFE](https://eprint.iacr.org/2015/064.pdf) or the numerous other fare exchange protocols.  And proxy reencryption obviously comes with major downsides.